### DISCUSSION TITLE
Slider de precios/cantidades

### DISCUSSION THREAD

**Discussion by GraceFinlay** [Telefonica/mistica-design#1018] [Proposals]
Buenas tardes,

Mi nombre es Grace y estoy trabajando en el proyecto de ecommerce global y os voy a indicar una serie de dudas relacionadas con Mística que nos han surgido en el desarrollo del proyecto.

# Contexto

El proyecto que está en marcha quiere incorporar unas mejoras en la tienda para que diferentes usuarios puedan comprar diferentes dispositivos. Estas mejoras que se quieren introducir son entre otras la mejora de los flujos de usuario y diferentes funcionalidades.

# Problema

Para este proyecto se requiere utilizar Mística y observamos que tenemos unas necesidades concretas en cuanto a componentes. Es importante obtener feedback de vuestra parte con el fin de entender cómo podríamos implementar el proyecto usando Mística y qué cubra las necesidades planteadas. En este caso necesitamos un slider  para la parte de los filtros que proporcione información de precios, cantidades y otros datos a los usuarios.

# Propuesta
![slider](https://user-images.githubusercontent.com/99174134/214302949-1817b6fd-f09d-406a-a22e-a0ef09646022.png)

Para el diseño de esté slider observamos que:

1. En Mística se contempla mostrar una cantidad pero en una nube.
2. Necesitaríamos mostrar de manera estática los datos de arriba a la vez que el usuario desplaza el slider.

Estamos abiertos a cualquier feedback que nos podáis dar para abordar el proyecto.

De antemano muchas gracias.

**Comment by yceballost** [comment]
Este componente lo tenemos en el roadmap pero la realidad es que ningún equipo hasta ahora (vosotros) lo había necesitado.

En la parte de diseño ya tenemos un poco de la definición, igual estaría bien darle una vuelta para ajustar algún detalle a vuestra necesidad, pero no lo veo complicado. Como por ejemplo, el tooltip (nube que comentas) que está en la definición actual entiendo que podría ser algo configurable en el componente.

Aquí podríamos seguir dos caminos:
1. Esperáis a que existe en Mística, no sabemos cuánto tiempo puede llevar, tendría que verlo con el equipo de ingeniería y hacer los ajustes en la parte de diseño
2. Trabajemos juntos las specs de diseño y vuestro equipo de desarrollo crea este componente y lo integra en mística-web haciendo una pull request al repo.

**Reply by yceballost** [reply]
https://github.com/Telefonica/mistica-design/issues/977
